File: courses/codes/java-class/Grade.java
import java.util.Scanner;

public class Grade {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		System.out.print("Enter numerical grade: ");
		int grade = Integer.parseInt(sc.nextLine());
		
		String letterGrade = "Invalid";
		
		if (grade >= 90 && grade <= 100) {
			letterGrade = "A";
		} else if (grade >= 80 && grade < 90) {
			letterGrade = "B";
		} else if (grade >= 70 && grade < 80) {
			letterGrade = "C";
		} else if (grade >= 60 && grade < 70) {
			letterGrade = "D";
		} else if (grade >= 0 && grade < 60) {
			letterGrade = "F";
		}
		
		// Alternative Solution
		if (grade > 100 || grade < 0) {
			letterGrade = "Invalid";
		} else if (grade >= 90) {
			letterGrade = "A";
		} else if (grade >= 80) {
			letterGrade = "B";
		} else if (grade >= 70) {
			letterGrade = "C";
		} else if (grade >= 60) {
			letterGrade = "D";
		} else if (grade >= 0) {
			letterGrade = "F";
		}
		
		
		
		System.out.println(letterGrade);
	}

}

File: courses/codes/python-class/grade.py
grade = int(input("Enter your grade: "))
letter_grade = "Invalid"

# if grade >= 90 and grade <= 100:
#     letter_grade = "A"
# elif grade >= 80 and grade < 90:
#     letter_grade = "B"
# elif grade >= 70 and grade < 80:
#     letter_grade = "C"
# elif grade >= 60 and grade < 70:
#     letter_grade = "D"
# elif grade >= 0 and grade < 60:
#     letter_grade = "F"

if grade > 100 or grade < 0:
    letter_grade = "Invalid"
elif grade >= 90:
    letter_grade = "A"
elif grade >= 80:
    letter_grade = "B"
elif grade >= 70:
    letter_grade = "C"
elif grade >= 60:
    letter_grade = "D"
elif grade >= 0:
    letter_grade = "F"


print(letter_grade)
# if 90 <= grade <= 100: